src/NCronJob/Execution/JobExecutor.cs

```diff
@@ -80,13 +77,21 @@ public async Task RunJob(JobRun run, CancellationToken stoppingToken)
     private IJob ResolveJob(IServiceProvider scopedServiceProvider, JobDefinition definition)
     {
         if (definition.Type == typeof(DynamicJobFactory))
-            return dynamicJobFactoryRegistry.GetJobInstance(scopedServiceProvider, definition);
+            return jobRegistry.GetDynamicJobInstance(scopedServiceProvider, definition);
 
         var job = scopedServiceProvider.GetService(definition.Type);
         if (job != null)
             return (IJob)job;
 
         LogUnregisteredJob(definition.Type);
+        // TODO: Maybe add a .UseNCronJob() extension method that would harvest the job definitions
```

Comment:
re #54

Reply:
Note: if you want me to drop this from the code, just say so.

Reply:
Preferable add the link to the issue - so we have the reference

Reply:
> Preferable add the link to the issue - so we have the reference

Done